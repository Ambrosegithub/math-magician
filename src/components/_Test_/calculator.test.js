import renderer from 'react-test-renderer';
import Calculator from '../calculator';

it('calculator render correctly', ()=>{
 const tree = renderer.create(<Calculator />).toJSON()
 expect(tree).toMatchSnapshot();
})