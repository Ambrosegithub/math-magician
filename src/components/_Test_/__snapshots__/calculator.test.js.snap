// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator render correctly 1`] = `
<section
  className="container"
>
  <h1
    className="calc-title"
  >
    Let's do some mathematics!
  </h1>
  <div
    className="calc-box"
  >
    <div
      className="screen"
    >
      <span
        className="output"
      >
        0
      </span>
    </div>
    <div
      className="buttons"
    >
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
         
        %
      </button>
      <button
        className="calc active"
        onClick={[Function]}
        type="button"
      >
        <span>
          รท
        </span>
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="calc active"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="calc active"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="calc active"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="zero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="calc"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="calc active"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</section>
`;
